app/app/src/main/java/tech/ziasvannes/safechat/presentation/components/MessageBubble.kt

```
@@ -0,0 +1,224 @@
+
+/**
+ * Formats a file size from bytes to a human-readable string
+ */
+private fun formatFileSize(size: Long): String {
+    val kb = 1024
+    val mb = kb * 1024
+    
+    return when {
+        size < kb -> "$size B"
+        size < mb -> "${size / kb} KB"
+        else -> String.format("%.1f MB", size.toFloat() / mb)
```

detekt (1.23.7)

[warning] String.format("%.1f MB", size.toFloat() / mb) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)